services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/model/CrtFileUpload.java

```diff
@@ -80,15 +88,24 @@ private ResumableFileUpload doPause() {
         }
 
         completionFuture.cancel(true);
-        // Upload hasn't started yet, or it's a single object upload
         if (token == null) {
+            // TODO - remove once CRT handles future completed exceptionally to return ResumeToken
```

Comment:
For this TODO do we have a follow up issue or task on CRT ?

Reply:
Yes ticket was cut to CRT